### ISSUE TITLE
[PATCH] Pod errors in perldiag.pod

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#2359]
Migrated from [rt.perl.org#3686](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=3686) (status was 'resolved')

Searchable as RT3686$

**Comment by p5pRT** [comment]
### From @abigail

#### Created by @abigail



This patch \(against perl@&#8203;6620\) fixes some pod errors in perldiag\.pod\.

Abigail


\*\*\* pod/perldiag\.pod\.orig    Sun Aug 13 14&#8203;:35&#8203;:36 2000
\-\-\- pod/perldiag\.pod    Tue Aug 15 00&#8203;:00&#8203;:00 2000
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 557\,566 \*\*\*\*
&nbsp;  characters and Perl was unable to create a unique filename during
&nbsp;  inplace editing with the B\<\-i> switch\.  The file was ignored\.
&nbsp;  
\! =item Can't do \{n\,m\} with n > m at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Minima must be less than or equal to maxima\. If you really want your
\! regexp to match something 0 times\, just put \{0\}\. The \<HERE\< shows in the
&nbsp;  regular expression about where the problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item Can't do setegid\!
\-\-\- 557\,566 \-\-\-\-
&nbsp;  characters and Perl was unable to create a unique filename during
&nbsp;  inplace editing with the B\<\-i> switch\.  The file was ignored\.
&nbsp;  
\! =item Can't do \{n\,m\} with n > m at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Minima must be less than or equal to maxima\. If you really want your
\! regexp to match something 0 times\, just put \{0\}\. The \<HEREE\<lt> shows in the
&nbsp;  regular expression about where the problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item Can't do setegid\!
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 1379\,1388 \*\*\*\*
&nbsp;  filehandles\.  Are you attempting to call flock\(\) on a dirhandle by the
&nbsp;  same name?
&nbsp;  
\! =item Quantifier follows nothing at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You started a regular expression with a quantifier\. Backslash it if you
\! meant it literally\. The \<HERE\< shows in the regular expression about where the
&nbsp;  problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item Format not terminated
\-\-\- 1379\,1388 \-\-\-\-
&nbsp;  filehandles\.  Are you attempting to call flock\(\) on a dirhandle by the
&nbsp;  same name?
&nbsp;  
\! =item Quantifier follows nothing at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You started a regular expression with a quantifier\. Backslash it if you
\! meant it literally\. The \<HEREE\<lt> shows in the regular expression about where the
&nbsp;  problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item Format not terminated
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 1629\,1638 \*\*\*\*
&nbsp;  internally\-\-subject to loss of precision errors in subsequent
&nbsp;  operations\.
&nbsp;  
\! =item Internal disaster at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(P\) Something went badly wrong in the regular expression parser\.
\! The \<HERE\< shows in the regular expression about where the problem was
&nbsp;  discovered\.
&nbsp;  
&nbsp;  
\-\-\- 1629\,1638 \-\-\-\-
&nbsp;  internally\-\-subject to loss of precision errors in subsequent
&nbsp;  operations\.
&nbsp;  
\! =item Internal disaster at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(P\) Something went badly wrong in the regular expression parser\.
\! The \<HEREE\<lt> shows in the regular expression about where the problem was
&nbsp;  discovered\.
&nbsp;  
&nbsp;  
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 1645\,1653 \*\*\*\*
&nbsp;  Perl is making a guess and treating this C\<exec> as a request to
&nbsp;  terminate the Perl script and execute the specified command\.
&nbsp;  
\! =item Internal urp at \<HERE\< in regex m/%s/
&nbsp;  
\! \(P\) Something went badly awry in the regular expression parser\. The \<HERE\<
&nbsp;  shows in the regular expression about where the problem was discovered\.
&nbsp;  
&nbsp;  
\-\-\- 1645\,1653 \-\-\-\-
&nbsp;  Perl is making a guess and treating this C\<exec> as a request to
&nbsp;  terminate the Perl script and execute the specified command\.
&nbsp;  
\! =item Internal urp at \<HEREE\<lt> in regex m/%s/
&nbsp;  
\! \(P\) Something went badly awry in the regular expression parser\. The \<HEREE\<lt>
&nbsp;  shows in the regular expression about where the problem was discovered\.
&nbsp;  
&nbsp;  
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 1753\,1762 \*\*\*\*
&nbsp;  values cannot be returned in subroutines used in lvalue context\.  See
&nbsp;  L\<perlsub/"Lvalue subroutines">\.
&nbsp;  
\! =item Lookbehind longer than %d not implemented at \<HERE\< in reges m/%s/
&nbsp;  
&nbsp;  \(F\) There is currently a limit on the length of string which lookbehind can
\! handle\. This restriction may be eased in a future release\. The \<HERE\< shows in
&nbsp;  the regular expression about where the problem was discovered\.
&nbsp;         
&nbsp;  =item Malformed PERLLIB\_PREFIX
\-\-\- 1753\,1762 \-\-\-\-
&nbsp;  values cannot be returned in subroutines used in lvalue context\.  See
&nbsp;  L\<perlsub/"Lvalue subroutines">\.
&nbsp;  
\! =item Lookbehind longer than %d not implemented at \<HEREE\<lt> in reges m/%s/
&nbsp;  
&nbsp;  \(F\) There is currently a limit on the length of string which lookbehind can
\! handle\. This restriction may be eased in a future release\. The \<HEREE\<lt> shows in
&nbsp;  the regular expression about where the problem was discovered\.
&nbsp;         
&nbsp;  =item Malformed PERLLIB\_PREFIX
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 1935\,1944 \*\*\*\*
&nbsp;  \(F\) You tried to do a read/write/send/recv operation with a buffer
&nbsp;  length that is less than 0\.  This is difficult to imagine\.
&nbsp;  
\! =item Nested quantifiers at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You can't quantify a quantifier without intervening parentheses\. So
\! things like \*\* or \+\* or ?\* are illegal\. The \<HERE\< shows in the regular
&nbsp;  expression about where the problem was discovered\.
&nbsp;  
&nbsp;  Note\, however\, that the minimal matching quantifiers\, C\<\*?>\, C\<\+?>\, and
\-\-\- 1935\,1944 \-\-\-\-
&nbsp;  \(F\) You tried to do a read/write/send/recv operation with a buffer
&nbsp;  length that is less than 0\.  This is difficult to imagine\.
&nbsp;  
\! =item Nested quantifiers at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You can't quantify a quantifier without intervening parentheses\. So
\! things like \*\* or \+\* or ?\* are illegal\. The \<HEREE\<lt> shows in the regular
&nbsp;  expression about where the problem was discovered\.
&nbsp;  
&nbsp;  Note\, however\, that the minimal matching quantifiers\, C\<\*?>\, C\<\+?>\, and
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 2704\,2716 \*\*\*\*
&nbsp;  \(S unsafe\) The subroutine being declared or defined had previously been
&nbsp;  declared or defined with a different function prototype\.
&nbsp;  
\! =item Quantifier in \{\,\} bigger than %d at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) There is currently a limit to the size of the min and max values of the
\! \{min\,max\} construct\. The \<HERE\< shows in the regular expression about where
&nbsp;  the problem was discovered\. See L\<perlre>\.
&nbsp;  
\! =item Quantifier unexpected on zero\-length expression at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(W regexp\) You applied a regular expression quantifier in a place where
&nbsp;  it makes no sense\, such as on a zero\-width assertion\.  Try putting the
\-\-\- 2704\,2716 \-\-\-\-
&nbsp;  \(S unsafe\) The subroutine being declared or defined had previously been
&nbsp;  declared or defined with a different function prototype\.
&nbsp;  
\! =item Quantifier in \{\,\} bigger than %d at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) There is currently a limit to the size of the min and max values of the
\! \{min\,max\} construct\. The \<HEREE\<lt> shows in the regular expression about where
&nbsp;  the problem was discovered\. See L\<perlre>\.
&nbsp;  
\! =item Quantifier unexpected on zero\-length expression at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(W regexp\) You applied a regular expression quantifier in a place where
&nbsp;  it makes no sense\, such as on a zero\-width assertion\.  Try putting the
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 2778\,2791 \*\*\*\*
&nbsp;  \(W internal\) The internal sv\_replace\(\) function was handed a new SV with
&nbsp;  a reference count of other than 1\.
&nbsp;  
\! =item Reference to nonexistant group at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You used something like C\<\\7> in your regular expression\, but there are
&nbsp;  not at least seven sets of capturing parentheses in the expression\. If you
&nbsp;  wanted to have the character with value 7 inserted into the regular expression\,
&nbsp;  prepend a zero to make the number at least two digits&#8203;: C\<\\07>
&nbsp;  
\! The \<HERE\< shows in the regular expression about where the problem was
&nbsp;  discovered\.
&nbsp;  
&nbsp;  =item regexp memory corruption
\-\-\- 2778\,2791 \-\-\-\-
&nbsp;  \(W internal\) The internal sv\_replace\(\) function was handed a new SV with
&nbsp;  a reference count of other than 1\.
&nbsp;  
\! =item Reference to nonexistant group at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You used something like C\<\\7> in your regular expression\, but there are
&nbsp;  not at least seven sets of capturing parentheses in the expression\. If you
&nbsp;  wanted to have the character with value 7 inserted into the regular expression\,
&nbsp;  prepend a zero to make the number at least two digits&#8203;: C\<\\07>
&nbsp;  
\! The \<HEREE\<lt> shows in the regular expression about where the problem was
&nbsp;  discovered\.
&nbsp;  
&nbsp;  =item regexp memory corruption
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 2890\,2898 \*\*\*\*
&nbsp;  \(W closed\) The socket you're sending to got itself closed sometime
&nbsp;  before now\.  Check your logic flow\.
&nbsp;  
\! =item Sequence \(? incomplete at \<HERE\< mark in regex m/%s/
&nbsp;  
\! \(F\) A regular expression ended with an incomplete extension \(?\. The \<HERE\<
&nbsp;  shows in the regular expression about where the problem was discovered\. See
&nbsp;  L\<perlre>\.
&nbsp;  
\-\-\- 2890\,2898 \-\-\-\-
&nbsp;  \(W closed\) The socket you're sending to got itself closed sometime
&nbsp;  before now\.  Check your logic flow\.
&nbsp;  
\! =item Sequence \(? incomplete at \<HEREE\<lt> mark in regex m/%s/
&nbsp;  
\! \(F\) A regular expression ended with an incomplete extension \(?\. The \<HEREE\<lt>
&nbsp;  shows in the regular expression about where the problem was discovered\. See
&nbsp;  L\<perlre>\.
&nbsp;  
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 2901\,2916 \*\*\*\*
&nbsp;  \(F\) If the contents of a \(?\{\.\.\.\}\) clause contains braces\, they must balance
&nbsp;  for Perl to properly detect the end of the clause\. See L\<perlre>\.
&nbsp;  
\! =item Sequence \(?%s\.\.\.\) not implemented at \<HERE\< mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) A proposed regular expression extension has the character reserved but
\! has not yet been written\. The \<HERE\< shows in the regular expression about
&nbsp;  where the problem was discovered\. See L\<perlre>\.
&nbsp;  
\! =item Sequence \(?%s\.\.\.\) not recognized at \<HERE\< mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You used a regular expression extension that doesn't make sense\.
\! The \<HERE\< shows in the regular expression about
&nbsp;  where the problem was discovered\. 
&nbsp;  See L\<perlre>\.
&nbsp;  
\-\-\- 2901\,2916 \-\-\-\-
&nbsp;  \(F\) If the contents of a \(?\{\.\.\.\}\) clause contains braces\, they must balance
&nbsp;  for Perl to properly detect the end of the clause\. See L\<perlre>\.
&nbsp;  
\! =item Sequence \(?%s\.\.\.\) not implemented at \<HEREE\<lt> mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) A proposed regular expression extension has the character reserved but
\! has not yet been written\. The \<HEREE\<lt> shows in the regular expression about
&nbsp;  where the problem was discovered\. See L\<perlre>\.
&nbsp;  
\! =item Sequence \(?%s\.\.\.\) not recognized at \<HEREE\<lt> mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) You used a regular expression extension that doesn't make sense\.
\! The \<HEREE\<lt> shows in the regular expression about
&nbsp;  where the problem was discovered\. 
&nbsp;  See L\<perlre>\.
&nbsp;  
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3098\,3104 \*\*\*\*
&nbsp;  \(F\) Your Perl was compiled with B\<\-D>SETUID\_SCRIPTS\_ARE\_SECURE\_NOW\, but
&nbsp;  a version of the setuid emulator somehow got run anyway\.
&nbsp;  
\! =item Switch \(?\(condition\)\.\.\. contains too many branches at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) A \(?\(condition\)if\-clause|else\-clause\) construct can have at most two
&nbsp;  branches \(the if\-clause and the else\-clause\)\. If you want one or both to
\-\-\- 3098\,3104 \-\-\-\-
&nbsp;  \(F\) Your Perl was compiled with B\<\-D>SETUID\_SCRIPTS\_ARE\_SECURE\_NOW\, but
&nbsp;  a version of the setuid emulator somehow got run anyway\.
&nbsp;  
\! =item Switch \(?\(condition\)\.\.\. contains too many branches at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) A \(?\(condition\)if\-clause|else\-clause\) construct can have at most two
&nbsp;  branches \(the if\-clause and the else\-clause\)\. If you want one or both to
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3107\,3119 \*\*\*\*
&nbsp;  
&nbsp;      \(?\(condition\)\(?&#8203;:this|that|other\)|else\-clause\)
&nbsp;  
\! The \<HERE\< shows in the regular expression about where the problem was
&nbsp;  discovered\. See L\<perlre>\.
&nbsp;  
\! =item Switch condition not recognized at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) If the argument to the \(?\(\.\.\.\)if\-clause|else\-clause\) construct is a
\! number\, it can be only a number\. The \<HERE\< shows in the regular expression
&nbsp;  about where the problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item switching effective %s is not implemented
\-\-\- 3107\,3119 \-\-\-\-
&nbsp;  
&nbsp;      \(?\(condition\)\(?&#8203;:this|that|other\)|else\-clause\)
&nbsp;  
\! The \<HEREE\<lt> shows in the regular expression about where the problem was
&nbsp;  discovered\. See L\<perlre>\.
&nbsp;  
\! =item Switch condition not recognized at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) If the argument to the \(?\(\.\.\.\)if\-clause|else\-clause\) construct is a
\! number\, it can be only a number\. The \<HEREE\<lt> shows in the regular expression
&nbsp;  about where the problem was discovered\. See L\<perlre>\.
&nbsp;  
&nbsp;  =item switching effective %s is not implemented
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3391\,3397 \*\*\*\*
&nbsp;  \(F\) There are no byte\-swapping functions for a machine with this byte
&nbsp;  order\.
&nbsp;  
\! =item Unknown switch condition \(?\(%\.2s at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) The condition of a \(?\(condition\)if\-clause|else\-clause\) construct is not
&nbsp;  known\. The condition may be lookaround \(the condition is true if the
\-\-\- 3391\,3397 \-\-\-\-
&nbsp;  \(F\) There are no byte\-swapping functions for a machine with this byte
&nbsp;  order\.
&nbsp;  
\! =item Unknown switch condition \(?\(%\.2s at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) The condition of a \(?\(condition\)if\-clause|else\-clause\) construct is not
&nbsp;  known\. The condition may be lookaround \(the condition is true if the
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3399\,3405 \*\*\*\*
&nbsp;  code evaluates to a true value\)\, or a number \(the condition is true if the
&nbsp;  set of capturing parentheses named by the number is defined\)\.
&nbsp;  
\! The \<HERE\< shows in the regular expression about where the problem was
&nbsp;  discovered\.  See L\<perlre>\.
&nbsp;  
&nbsp;  =item Unknown open\(\) mode '%s'
\-\-\- 3399\,3405 \-\-\-\-
&nbsp;  code evaluates to a true value\)\, or a number \(the condition is true if the
&nbsp;  set of capturing parentheses named by the number is defined\)\.
&nbsp;  
\! The \<HEREE\<lt> shows in the regular expression about where the problem was
&nbsp;  discovered\.  See L\<perlre>\.
&nbsp;  
&nbsp;  =item Unknown open\(\) mode '%s'
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3415\,3428 \*\*\*\*
&nbsp;  data Perl expected\.  Someone's very confused\, or perhaps trying to
&nbsp;  subvert Perl's population of %ENV for nefarious purposes\.
&nbsp;  
\! =item unmatched \[ at \<HERE\< mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) The brackets around a character class must match\. If you wish to
&nbsp;  include a closing bracket in a character class\, backslash it or put it
\! first\. See L\<perlre>\. The \<HERE\< shows in the regular expression about
&nbsp;  where the escape was discovered\.
&nbsp;  
\! =item unmatched \( in regexp at \<HERE\< mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Unbackslashed parentheses must always be balanced in regular
&nbsp;  expressions\.  If you're a vi user\, the % key is valuable for finding the
\-\-\- 3415\,3428 \-\-\-\-
&nbsp;  data Perl expected\.  Someone's very confused\, or perhaps trying to
&nbsp;  subvert Perl's population of %ENV for nefarious purposes\.
&nbsp;  
\! =item unmatched \[ at \<HEREE\<lt> mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) The brackets around a character class must match\. If you wish to
&nbsp;  include a closing bracket in a character class\, backslash it or put it
\! first\. See L\<perlre>\. The \<HEREE\<lt> shows in the regular expression about
&nbsp;  where the escape was discovered\.
&nbsp;  
\! =item unmatched \( in regexp at \<HEREE\<lt> mark in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Unbackslashed parentheses must always be balanced in regular
&nbsp;  expressions\.  If you're a vi user\, the % key is valuable for finding the
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3454\,3465 \*\*\*\*
&nbsp;  recognized by Perl inside character classes\.  The character was
&nbsp;  understood literally\.
&nbsp;  
\! =item Unrecognized escape \\\\%c passed through at \<HERE\< in m/%s/
&nbsp;  
&nbsp;  \(W regexp\) You used a backslash\-character combination which is not
&nbsp;  recognized by Perl\. This combination appears in an interpolated variable or
&nbsp;  a C\<'>\-delimited regular expression\. The character was understood
\! literally\. The \<HERE\< shows in the regular expression about where the escape
&nbsp;  was discovered\.
&nbsp;  
&nbsp;  
\-\-\- 3454\,3465 \-\-\-\-
&nbsp;  recognized by Perl inside character classes\.  The character was
&nbsp;  understood literally\.
&nbsp;  
\! =item Unrecognized escape \\\\%c passed through at \<HEREE\<lt> in m/%s/
&nbsp;  
&nbsp;  \(W regexp\) You used a backslash\-character combination which is not
&nbsp;  recognized by Perl\. This combination appears in an interpolated variable or
&nbsp;  a C\<'>\-delimited regular expression\. The character was understood
\! literally\. The \<HEREE\<lt> shows in the regular expression about where the escape
&nbsp;  was discovered\.
&nbsp;  
&nbsp;  
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 3748\,3757 \*\*\*\*
&nbsp;  reference variables in outer subroutines are called or referenced\, they
&nbsp;  are automatically rebound to the current values of such variables\.
&nbsp;  
\! =item Variable length lookbehind not implemented at \<HERE\< in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Lookbehind is allowed only for subexpressions whose length is fixed and
\! known at compile time\. The \<HERE\< shows in the regular expression about where
&nbsp;  the problem was discovered\.
&nbsp;  
&nbsp;  =item Version number must be a constant number
\-\-\- 3748\,3757 \-\-\-\-
&nbsp;  reference variables in outer subroutines are called or referenced\, they
&nbsp;  are automatically rebound to the current values of such variables\.
&nbsp;  
\! =item Variable length lookbehind not implemented at \<HEREE\<lt> in regex m/%s/
&nbsp;  
&nbsp;  \(F\) Lookbehind is allowed only for subexpressions whose length is fixed and
\! known at compile time\. The \<HEREE\<lt> shows in the regular expression about where
&nbsp;  the problem was discovered\.
&nbsp;  
&nbsp;  =item Version number must be a constant number

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=low

Site configuration information for perl v5.6.0:

Configured by abigail at Mon Aug 14 23:07:00 EDT 2000.

Summary of my perl5 (revision 5.0 version 6 subversion 0) configuration:
  Platform:
    osname=linux, osvers=2.2.13, archname=i686-linux-64int
    uname='linux alexandra 2.2.13 #5 tue feb 8 15:37:54 est 2000 i686 unknown '
    config_args='-Dprefix=/opt/devperl -d -Doptimize=-g -Dusemorebits'
    hint=recommended, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=define 
    use64bitint=define use64bitall=undef uselongdouble=define usesocks=undef
  Compiler:
    cc='cc', optimize='-g', gccversion=2.95.2 19991024 (release), gccosandvers=
    cppflags='-DDEBUGGING -fno-strict-aliasing -I/usr/local/include'
    ccflags ='-DDEBUGGING -fno-strict-aliasing -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
    stdchar='char', d_stdstdio=define, usevfork=false
    intsize=4, longsize=4, ptrsize=4, doublesize=8
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long long', ivsize=8, nvtype='long double', nvsize=12, Off_t='off_t', lseeksize=8
    alignbytes=4, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib
    libs=-lnsl -lndbm -lgdbm -ldbm -ldb -ldl -lm -lc -lposix -lcrypt -lutil
    libc=/lib/libc-2.1.2.so, so=so, useshrplib=false, libperl=libperl.a
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-rdynamic'
    cccdlflags='-fpic', lddlflags='-shared -L/usr/local/lib'

Locally applied patches:
    


@INC for perl v5.6.0:
    /home/abigail/Perl
    /home/abigail/Sybase
    /opt/devperl/lib/5.6.0/i686-linux-64int
    /opt/devperl/lib/5.6.0
    /opt/devperl/lib/site_perl/5.6.0/i686-linux-64int
    /opt/devperl/lib/site_perl/5.6.0
    /opt/devperl/lib/site_perl/5.5.650
    /opt/devperl/lib/site_perl/5.5.640
    /opt/devperl/lib/site_perl/5.00563
    /opt/devperl/lib/site_perl/5.00562
    /opt/devperl/lib/site_perl/5.00561
    /opt/devperl/lib/site_perl/5.00558
    /opt/devperl/lib/site_perl/5.00557
    /opt/devperl/lib/site_perl
    .


Environment for perl v5.6.0:
    HOME=/home/abigail
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH=/home/abigail/Lib:/usr/local/lib:/usr/lib:/lib:/usr/X11R6/lib:/opt/tcl/lib:/opt/tk/lib/tk8.0
    LOGDIR (unset)
    PATH=/home/abigail/Bin:/opt/perl/bin:/opt/tcl/bin:/opt/tk/bin:/usr/local/bin:/usr/local/X11/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/X11R6/bin:/usr/games:/opt/povray/bin:/opt/teTeX/bin/i686-pc-linux-gnu:/opt/python/bin
    PERL5LIB=/home/abigail/Perl:/home/abigail/Sybase
    PERLDIR=/opt/perl
    PERL_BADLANG (unset)
    SHELL=/bin/bash


```
</details>

**Comment by p5pRT** [comment]
### From The RT System itself

closed&#8203;: @&#8203;7385 seems to have these changes\, but in a different style\.
